Modules/Product/Http/Services/ProductService.php

```diff
@@ -329,7 +438,7 @@ public function create($request): array
             ->update([
                 'feature_key' => 1
             ]);
-        $productImage = ProductImage::where('product_id', $lastInsertId)->where('image_sort', $featuredKey)->first();
+        $productImage = Product::find($lastInsertId)->productImages()->where('image_sort', $featuredKey)->first();
         $featuredImage = $productImage->images;
         Product::where('id', $lastInsertId)
             ->update([
```

Comment:
You can reduce these two queries into one using relations.
Use it like this:
$product = Product::with('productImages')->where('id',$lastInsertId)->first();
$productImage = $prodcut->productImages()->where('image_sort', $featuredKey)->first();